package.json

```diff
@@ -5,7 +5,13 @@
   "dependencies": {
     "react": "^16.8.6",
     "react-dom": "^16.8.6",
-    "react-scripts": "3.0.1"
+    "react-redux": "^7.1.0",
```

Comment:
we're not using redux anywhere, please remove all unnecessary dependencies including `react-router-dom` and whatever else